magentoerpconnect/magento_model.py

```diff
@@ -364,6 +370,15 @@ class MagentoConfigSpecializer(models.AbstractModel):
         'The value can also be specified on website or the store or the '
         'store view.'
     )
+    specific_warehouse_id = fields.Many2one(
+        comodel_name='stock.warehouse',
+        string='Specific warehouse',
+        help='If specified, this warehouse will be used to load fill the '
+        'field warehouse (and company) on the sale order created by the '
+        'connector.'
```

Comment:
This is not working in my setup. We get another warehouse from the same company. In fact, I cannot seem to find any place (in this patch or elsewhere) where the Magento warehouse setting is used to assign a warehouse to the sale orders imported from Magento. The Magento documentation does not even claim that the warehouse setting is used as such, but says the warehouse is used to gather the available product quantities instead (http://odoo-magento-connector.com/howto/configure_warehouse.html). Can you confirm that this setting is actually ignored for sale orders created from the Magento import?